Jenkinsfile

```diff
@@ -1220,6 +1220,26 @@ pipeline{
                             }
                             }
               }
+         stage('rh7_cluster_11.0.0'){
+            agent { label 'dhfLinuxAgent'}
+            steps{
+             timeout(time: 5,  unit: 'HOURS'){
+               catchError(buildResult: 'SUCCESS', catchInterruptions: true, stageResult: 'FAILURE'){dhflinuxTests("11.0.0","Release")}
```

Comment:
for 11.0 nightly we need to do "11.0","Latest" instead of "11.0.0","Release"

Reply:
@SameeraPriyathamTadikonda 11.0-Nightly is already added to nightly, expectation is to add ML latest released version

Reply:
Okay, Looks good.